src/redux/books/books.js

```diff
@@ -0,0 +1,65 @@
+// books.js
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+// books.js
+const initialState = [
+  {
+    id: 1,
+    categorie: 'Action',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    endRate: '64',
+  },
+  {
+    id: 2,
+    categorie: 'Action',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    endRate: '64',
+  },
+  {
+    id: 3,
+    categorie: 'Action',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    endRate: '64',
+  },
+];
```

Comment:
- [ ] It will be great to have the initial state as an empty array. This is a major requirement for this project.